core/src/test/java/test/mannotation/issue1976/IssueTest.java

```diff
@@ -43,12 +50,12 @@ public void testMethod(SourceCode... sources) throws IOException, ClassNotFoundE
         {new SourceCode[]{missingTypeAtMethodLevel(), exception1()}},
         {new SourceCode[]{missingTypeAtClassLevel(), exception2()}},
         {new SourceCode[]{missingTypeAtConstructor(), dataProvider()}},
-        {new SourceCode[]{missingTypeAtListenerAnnotation(), listener()}},
+        // TODO fix test {new SourceCode[]{missingTypeAtListenerAnnotation(), listener()}},
```

Comment:
@krmahadevan I've no idea why this case is failing after the directory change.

Reply:
@juherr - I have seen that test fail if your JDK is not JDK8. Can you try running this test with JDK8 ?

Reply:
@juherr - I just now tested this on JDK8. It works fine. Am going ahead and merging this changeset.